spec/unit/mutest/cli_spec.rb

```diff
@@ -151,6 +151,19 @@
         let(:expected_integration) { Mutest::Integration::Rspec }
       end
 
+      context 'when specifying the default null integration explicitely' do
+        let(:flags) { %w[--use null] }
+        let(:expected_integration) { Mutest::Integration::Null }
+
+        before do
+          expect(Kernel).to receive(:require)
+            .with('mutest/integration/null')
+            .and_call_original
```

Comment:
You said before

> I thought about this some more and I think the best solution is to split the tests: Have one test that tests the process (basically how it was), and one test that checks existence of the file for all supported integrations.

Is this still your intention? I imagined that included deleting the `and_call_original`.